client/src/components/Logo.js

```diff
@@ -5,24 +5,24 @@ import styled from 'styled-components';
 import logo from '../styles/assets/logo.svg';
 
 const Wrapper = styled.a.attrs({
-    className: 'navbar-brand',
+  className: 'navbar-brand',
 })``;
 
 class Logo extends Component {
```

Comment:
Just checking -- do we want to use functional or class based components? I generally prefer the former, but it's not a huge deal if you'd prefer not to update them.

Reply:
I wan't sure what style of react do the fellows learn, so I didn't update class components ...

Reply:
Honestly, I think it's anything and everything lol

I would suggest having examples of different ways of writing components and leaving a comment above their definitions to briefly explain them (or at least what that kind of component is called). Could also be useful to link to documentation/articles about different types of components but you may also run the risk of having dead links lying around at some point.